lib/api/api.js

```diff
@@ -87,9 +89,11 @@ const api = {
     bucketDelete,
     bucketGet,
     bucketGetACL,
+    bucketGetVersioning,
```

Comment:
I think you mean the bucket owner can do so with respect to ACL's.  This doesn't mean that a user under the owner account does not have the right if a proper policy is attached.  I just tested with a user in real AWS and users can put and get bucket versioning.  That being said we can add an issue to integration to add this test after this PR.